client/status/v2_test.go

```diff
@@ -224,7 +224,7 @@ func TestFromStatusV2(t *testing.T) {
 		}
 
 		randomError := errors.New("garbage")
-		errFromStatus := apistatus.ErrFromStatus(st)
+		errFromStatus, _ := st.(error)
```

Comment:
All our statuses already implements `error` interface

Reply:
Then it's just `errFromStatus := st.(error)`.

Reply:
Updated this part of code. In our test-cases no all `st` are apistatus structures. Some of them is int, string or bool

Reply:
After one more update, the code is irrelevant.